src/CLI/clitree/cli-xml/image.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This is only a CLI skeleton at this point, with no back-end support. Is this going to be OK to merge to the SONiC community?